tensorflow/compiler/tf2tensorrt/convert/trt_optimization_pass.cc

```diff
@@ -367,12 +378,27 @@ Status TRTOptimizationPass::Optimize(grappler::Cluster* cluster,
   }
 
   if (use_calibration_ && precision_mode_ != TrtPrecisionMode::INT8) {
-    VLOG(1) << "Calibration with FP32 or FP16 is not implemented. "
-            << "Falling back to use_calibration = False."
-            << "Note that the default value of use_calibration is True.";
+    LOG(WARNING) << "Calibration with FP32 or FP16 is not implemented. "
+                 << "Falling back to use_calibration = False."
+                 << "Note that the default value of use_calibration is True.";
     use_calibration_ = false;
   }
 
+  const bool use_explicit_precision = GraphDefHasQDQNodes(item.graph);
```

Comment:
I have two questions regarding this:
(1) we don't consider the TensorRT version to enable use_explicit_precision, this looks wrong to me.
For example, a user has a model that uses QuantizeAndDequantizeV2 + TensorRT 7. It was working but now with this PR, he sees an error produced by TensorRT, similar to that in quantization_ops_test.cc  line 435 & 440. Am I right here?
(2)  A user has a model that uses all the OPs in kQuantizationOpNames  + TensorRT8. It was working fine. But now with your change, we only convert one out of the four OPs in kQuantizationOpNames . Will this be a problem?

Reply:
OK yes, I let me disable this for TRT < 8.0. We can use TRT 7 for very limited use cases, but likely EP mode won't be useful in the context of TF-TRT with TRT7 at this time.

Reply:
fixed